thirdparty/build-thirdparty.sh

```diff
@@ -1239,6 +1239,20 @@ build_clucene() {
     fi
 }
 
+# avx2ki
+build_avx2ki() {
+    check_if_source_exist $AVX2KI_SOURCE
+    cd $TP_SOURCE_DIR/$AVX2KI_SOURCE
+    rpm2cpio boostkit-ksl-2.0.0-1.aarch64.rpm | cpio -id
+    mv usr/local/ksl $TP_INSTALL_DIR
+    cp $TP_INSTALL_DIR/ksl/lib/libavx2neon.so.2.0.0 $TP_INSTALL_DIR/lib/libavx2neon.so
+
+     # patch avx2ki/operatoroverload.h
+     patch $TP_INSTALL_DIR/ksl/include/operatoroverload.h -p1 < $TP_PATCH_DIR/avx2ki-2.0.0-operation.patch
```

Comment:
not sure if this is a good idea, we are patching the header without rebuilding the binary, need carefully evaluate the risk

Reply:
We have evaluated that the changes only work in the header, and do not affect the behaviors in the binary.